doc/api/process.md

```diff
@@ -1509,6 +1509,20 @@ tarball.
   - `'Argon'` for the 4.x LTS line beginning with 4.2.0.
   - `'Boron'` for the 6.x LTS line beginning with 6.9.0.
   - `'Carbon'` for the 8.x LTS line beginning with 8.9.1.
+* `majorVersion` {number} The major version of Node.js.
+* `minorVersion` {number} The minor version of Node.js.
+* `patchVersion` {number} The patch version of Node.js.
+* `prereleaseTag` {string} The SemVer pre-release tag for Node.js.
+* `compareVersion` {function} Perform a SemVer comparison to the release
+  version.
+    * `major` {string | number} The major version to compare or a string
+      containing the entire version.
+    * `minor` {number} The minor version number to compare.
+    * `patch` {number} The patch version to compare.
+    * `tag` {string} The pre-release tag to compare.
+    * Returns: {number} `-1` if the given version is lower than the release
+      version, `0` if the given version matches the process version, and `1`
+      if the given version is greater than the release version.
```

Comment:
I already explained in #19574 why this design is problematic:

> The signum of the result of `compareVersion` seems counter-intuitive to me. In most (if not all) frameworks, a comparator returns a negative result if the first argument (or `this`) is considered to be "before" the second argument, and a positive result if the first argument (or `this`) is considered to be "after" the second argument (see e.g. [java.lang.Comparable](https://docs.oracle.com/javase/7/docs/api/java/lang/Comparable.html#compareTo(T)) for a single-argument comparison). `compareVersion` contradicts this well-established norm. I would expect `compareVersion(10, 0, 0) >= 0` to return whether this release is v10 or newer, which follows the norm and is intuitive, but https://github.com/nodejs/node/pull/19438 implements the opposite.

In one of your earlier revisions of your comment in that PR, you seemed to agree with me. Is there are reason to stick to the old design?